UnitsNet.Tests/CustomCode/SpeedTests.cs

```diff
@@ -64,6 +64,10 @@ public class SpeedTests : SpeedTestsBase
 
         protected override double KilometersPerMinutesInOneMeterPerSecond => 0.06;
 
+        protected override double CentimetersPerHourInOneMeterPerSecond => 3600*100;
```

Comment:
I would prefer if the tests did not perform calculations, but rather write the constant value directly. 

I think it is an important exercise that the values here are not derived or calculated, but rather looked up in a source when possible, at least as a rule of thumb.